README.md

```diff
@@ -0,0 +1,161 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="cat-black-face.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio: setup and mobile first</b></h3>
```

Comment:
- Please remove this title as this is not a Portfolio project, it is important to do this so as to keep your readme professional 😄

Reply:
Done👍😊